Sources/StreamChatUI/Utils/NukeImageProcessor.swift

```diff
@@ -63,11 +63,13 @@ extension ImageProcessors {
             return size
         }
 
+        private let url: URL
         private let sizeProvider: () -> CGSize
-        
+
         /// Initializes the processor with size providing closure.
         /// - Parameter sizeProvider: Closure to obtain size after the image is loaded.
-        public init(sizeProvider: @escaping () -> CGSize) {
+        public init(url: URL, sizeProvider: @escaping () -> CGSize) {
```

Comment:
Can we add documentation to the new URL parameter?

Reply:
Also, we changed the init of this component, how there are no changes in the rest of the codebase? 🤔

Reply:
Maybe you missed things: https://github.com/GetStream/stream-chat-swift/pull/1747/commits/16b06b0b2110afd136330aa9f6455cd1060f78ba

I will add docu, yeah

Reply:
I see... I thought this was the init from the `NukeImageLoader`, sometimes the diffing on GitHub is not clear 😄 

I didn't even know this LateResize was from our SDK... It is odd that this util is separate from the `ImageLoading` folder and I think it should even be part of the `NukeImageLoader` file. Anyways this is kinda a breaking change since it is a public init, and we are adding a new required argument 🤔 I'm sure it is not a big deal and nobody is using this, but in practice this is a breaking change :/

Reply:
Good point! Fixed here: https://github.com/GetStream/stream-chat-swift/pull/1747/commits/7fc2d9c2864b3e658bdddfdea072f522b5ea73e2